### ISSUE TITLE
validation set and hyperparameters tuning

### ISSUE THREAD

**Issue by OussamaBellalah** [ZhengyaoJiang/PGPortfolio#131]
Hello,
I have found it difficult searching through the code for the validation set ( or the cross validation set as referred to in the paper) reading through datamatrices.py I came to realize that the author is only subdividing the sets into training and test set, no mention for validation is done. The author expects the reader to tune in the hyperparameters based on the validation set, especially if the hyperparameters should not remain the same if we change the time window, so my question is the following:

How are we supposed to tune in the hyperparameters?

thanks in advance
